### PR TITLE
Listing directories

### PR DESCRIPTION
**Pull request by danielrcoura**
Related to issue #11
- Created a new controller file directories.js
- Created a new route file directories.js
- Renamed the controller file index.js to login.js
- Renamed the route file index.js to login.js
- Added middleware in app.js to verify if user is login

### CODE REVIEW

File: routes/servers.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = function(app) {
+    var servers = app.controllers.servers;
+
+    app.route('/api/servers').get(servers.get);
+}
```
Comment (dfquaresma) [inline routes/servers.js]: Same thing about newline

File: controllers/directories.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+const Client = require('ssh2-sftp-client');
```
[the remark this answers is not in the document]
Comment (danielrcoura) [reply controllers/directories.js]: As it is an instance of a module, I think it's best to be lowercase. In several tutorials the 'var' command is used to declare, I saw one that used 'const' and I found it more suitable, but I can switch to the most used one.

File: routes/directories.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = function(app) {
+    var directories = app.controllers.directories;
+
+    app.route('/api/directories').get(directories.get);
+}
```
[the remark this answers is not in the document]
Comment (danielrcoura) [reply routes/directories.js]: I'll do this.

File: controllers/directories.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dfquaresma) [reply controllers/directories.js]: @danielrcoura do it only if you think it's the best. It was just a suggestion. 
I thought "const" should create a constant variable and in java constants are uppercase. I don't know how it's in JS. 
I believe that we should do as you said. Following the most used.

[the remark this answers is not in the document]
Comment (danielrcoura) [reply controllers/directories.js]: I've switched to "var", it's more common.

**Pull request merged**